extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -168,16 +168,24 @@ it lives on the non-AMP document that is iframed.
 If you're using a common video framework like JwPlayer or Video.js, you can
 simply call `listenTo` for a basic integration:
 
-##### For JwPlayer:
+##### For JwPlayer
+
+Pass in your [`jwplayer` instance object](https://developer.jwplayer.com/jw-player/docs/javascript-api-reference/)
+through the signature `ampIntegration.listenTo('jwplayer', myJwplayer)`. The `ampIntegration` object then knows how
+to setup the player through the instance API.
```

Comment:
Worth throwing in a `{% call callout('Important', type='caution') %}` here maybe?

Reply:
Caution about what? :P

Reply:
(added a general `note` above.)